getting_started/introduction/introduction_to_godot.rst

```diff
@@ -0,0 +1,126 @@
+Introduction to Godot
+=====================
+
+This article is here to help you figure out whether Godot might be a good fit
+for you. We will introduce some broad features of the engine to give you a feel
+for what you can achieve with it and answer questions such as "what do I need to
+know to get started?".
+
+This is by no means an exhaustive overview. We will introduce many more features
+in this getting started series.
+
+What is Godot?
+--------------
+
+Godot is a general-purpose 2D and 3D game engine designed to support all sorts
+of projects. You can use it to create games or applications you can then release
+on desktop or mobile, as well as on the web.
+
+You can also create console games with it, although you either need strong
+programming skills or a developer to port the game for you.
+
+.. note:: The Godot team can't provide an open-source console export due to the
+          licensing terms imposed by console manufacturers. Regardless of the
+          engine you use, though, releasing games on consoles is always a lot of
+          work. You can read more on that here:
+
+What can the engine do?
+-----------------------
+
+Godot was initially developed in-house by an Argentinan game studio. Its
+development started in 2001, and the engine was rewritten and improved
+tremendously since its open-source release in 2014.
+
+Some examples of games created with Godot include Ex-Zodiac and Helms of Fury.
+
+.. image:: img/introduction_ex_zodiac.png
+
+.. image:: img/introduction_helms_of_fury.jpg
+
+As for applications, the open-source pixel art drawing program Pixelorama is
+powered by Godot, and so is the voxel RPG creator RPG in a box.
+
+.. image:: img/introduction_rpg_in_a_box.png
+
+You can find many more examples in the official showcase videos:
+
+- `April 2020 desktop and console showcase`_
+- `April 2020 mobile showcase`_
+
+How does it work and look?
+--------------------------
+
+Godot comes with a fully-fledged game editor with integrated tools to answer the
+most common needs. It includes a code editor, an animation editor, a tilemap
+editor, a shader editor, a debugger, a profiler, and more.
+
+.. image:: img/introduction_editor.png
+
+The team strives to offer a complete game creation package with a consistent
```

Comment:
This is misleading (or wrong). Godot is not a complete game creation tool unto itself, but rather another free component of a fully functioning open source game development ecosystem made useful by an image editor like GIMP or Krita, 3D modeling software such as Blender, and more.

Reply:
Meant it in the sense of full-featured game engine with editors for most tasks (script editor, tileset, anim, anim tree, profiler, etc.). Editing this now.